### PR TITLE
Add filter transform to dev

### PR DESCRIPTION
**Pull request by dmt0**
Implemented filter transforms

### CODE REVIEW

File: src/default_panels/GraphTransformsPanelDev.js [hunk 1]
```diff
@@ -71,21 +72,8 @@ const GraphTransformsPanel = (props, {localize: _}) => {
 
         <DataSelector label={_('By')} attr="groups" />
 
-        <DataSelector label={_('Filter')} attr="target" />
```
Comment (nicolaskruchten) [inline src/default_panels/GraphTransformsPanelDev.js]: I think you still need `target` right?

File: src/components/fields/FilterOperation.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+      operations[key].map(o => o.value).indexOf(operator) !== -1
+    ) {
+      op = key;
+      break;
+    }
+  }
+  return op;
+};
+
+class UnconnectedFilterOperation extends Component {
```
Comment (VeraZab) [inline src/components/fields/FilterOperation.js]: would you put this component in `components/fields` like for all our other connected components :)

File: src/default_panels/GraphTransformsPanelDev.js [hunk 3]
```diff
@@ -72,6 +72,7 @@ const GraphTransformsPanel = (props, {localize: _}) => {
 
         <DataSelector label={_('By')} attr="groups" />
 
+        <DataSelector label={_('Target')} attr="target" />
```
Comment (VeraZab) [inline src/default_panels/GraphTransformsPanelDev.js]: I would have put `By` as a label here as well

**Comment by dmt0** [comment]
Basically visibility rule for DataSelector now becomes: always show, unless it's in transforms

**Comment by VeraZab** [comment]
afterwards 💃

**Pull request merged**